api/v1beta1/flinkcluster_validate.go

```diff
@@ -406,7 +406,11 @@ func (v *Validator) validateJob(jobSpec *JobSpec) error {
 	if *jobSpec.Parallelism < 1 {
 		return fmt.Errorf("job parallelism must be >= 1")
 	}
-
+	if jobSpec.ParallelismPerTaskManager != nil    {
+		if  *jobSpec.ParallelismPerTaskManager * tmSpec.Replicas != *jobSpec.Parallelism {
```

Comment:
Currently, if not specified, the default value of `jobSpec.Parallelism` is 1, but I think if we make `jobSpec.ParallelismPerTaskManager` and `jobSpec.Parallelism` mutually exclusive, it will be less confusing for users, but we need to document it well.

Reply:
Okay agreed on this